stdlib/stdlib.mli

```diff
@@ -13,15 +13,21 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(** The initially opened module.
+(** The OCaml Standard library.
 
-   This module provides the basic operations over the built-in types
-   (numbers, booleans, byte sequences, strings, exceptions, references,
-   lists, arrays, input-output channels, ...).
+    This module is automatically opened at the beginning of each
+    compilation. All components of this module can therefore be
+    referred by their short name, without prefixing them by [Stdlib].
+*)
+
+module Pervasives : sig
+(** Pervasives operations.
+
+    This module provides the basic operations over the built-in types
+    (numbers, booleans, byte sequences, strings, exceptions, references,
+    lists, arrays, input-output channels, ...).
 
-   This module is automatically opened at the beginning of each compilation.
-   All components of this module can therefore be referred by their short
-   name, without prefixing them by [Pervasives].
+    This module is included in the toplevel {!Stdlib} module.
```

Comment:
I'm not convinced by this change (at this juncture). Possibly revert to the old paragraph? If not, something needs to be done to remove the {!Stdlib} cross-reference when building with ocamldoc, as it's causing a warning.